src/py/flwr/server/fleet/grpc_bidi/ins_scheduler.py

```diff
@@ -162,6 +163,7 @@ def _call_client_proxy(
         evaluate_res: EvaluateRes = client_proxy.evaluate(
             ins=evaluate_ins,
             timeout=timeout,
+            group_id=None,
```

Comment:
I'm not so sure about this part but, shouldn't there be some `group_id` passed in the `evaluate_ins` before that can be used to set `group_id` in the result? the same applies to `fit_ins` a couple of lines above.

Reply:
That would be ideal, but, from what I was able to see, there is no way from the `ins_scheduler` to get the round number. Also, from what I understood, this piece of code is still present only for backwards compatibility and will be removed in the future, is that correct @danieljanes ?